tasks/support/gem_base64.rb

```diff
@@ -0,0 +1,23 @@
+require "base64"
+require "rubygems/package"
+
+class GemBase64
+
```

Comment:
Nothing in the `tasks` dir is going to be included as part of the gem when it is packaged, this is just support for building the gem, or developer libs, not part of the gem itself.